packages/neuron-ui/src/components/NervosDAORecord/index.tsx

```diff
@@ -137,7 +137,9 @@ export const DAORecord = ({
         break
       }
       case CellStatus.Deposited: {
-        if (leftHours) {
+        if (!compensationEndEpochValue) {
+          message = ''
```

Comment:
@Danie0918 we can add a easy-to-understand message for this case

Reply:
Is this the tip in the calculation?

If so, here's a tip to add `Calculating the next compensation cycle`

Reply:
> Is this the tip in the calculation?
> 
> If so, here's a tip to add `Calculating the next compensation cycle`

In some cases, the data for calculation are missing, e.g. fail to get the tip block number from the CKB node, the remaining time cannot be displayed until the tip block number is available. Users should be informed to wait for the calculation instead of an incorrectly estimated remaining time.

Reply:
cc @devchenyan

Reply:
Maybe it's `the current compensation cycle`, the remaining time to the next compensation cycle.

Reply:
That would be `Calculating the compensation cycle`.

Reply:
> That would be `Calculating the compensation cycle`.

LGTM

Reply:
> That would be `Calculating the compensation cycle`.

![image](https://github.com/user-attachments/assets/a22a25be-0948-4c8c-92c7-c52f322a0199)